packages/browser/src/login/oidc/incomingRedirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -105,6 +105,14 @@ export class AuthCodeRedirectHandler implements IIncomingRedirectHandler {
       this.issuerConfigFetcher
     );
 
+    const iss = url.searchParams.get("iss") as string;
```

Comment:
```suggestion
    const iss = url.searchParams.get("iss");
```
This cast is made in other checks because it matches the `canHandle` conditions, so we know it has already been tested to be non-null, which isn't the case of the `iss` parameter, which is optional so not part of the `canHandle` check for the auth code flow.